client/components/empty-content/index.js

```diff
@@ -0,0 +1,145 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { H } from 'layout/section';
+import ImageAsset from 'components/image-asset';
+
+/**
+ * A component to be used when there is no data to show.
+ * It can be used as an opportunity to provide explanation or guidance to help a user progress.
+ */
+class EmptyContent extends Component {
+	renderIllustration() {
+		const { illustrationWidth, illustrationHeight, illustration } = this.props;
+		return (
+			<ImageAsset
+				alt=""
+				src={ illustration }
+				width={ illustrationWidth }
+				height={ illustrationHeight }
+				className="woocommerce-empty-content__illustration"
+			/>
+		);
+	}
+
+	renderActionButtons( type ) {
+		const actionLabel =
+			'secondary' === type ? this.props.secondaryActionLabel : this.props.actionLabel;
+		const actionURL = 'secondary' === type ? this.props.secondaryActionURL : this.props.actionURL;
+		const actionCallback =
+			'secondary' === type ? this.props.secondaryActionCallback : this.props.actionCallback;
+
+		if ( actionURL && actionCallback ) {
+			return (
+				<Button
+					className="woocommerce-empty-content__action"
+					isPrimary
```

Comment:
Should this depend on whether it's `primary` or `secondary`?

Reply:
Oops yeah. I combined some logic after Paul's review and missed that. Thanks for catching!